package.json

```diff
@@ -12,8 +12,7 @@
     "serve": "parcel index.html",
```

Comment:
Now that we have the `dev.js` script, I guess we don't really need this `serve` thing anymore (if we ever needed it)

Reply:
Yeah it is from an old test.